akka-bench-jmh/build.sbt

```diff
@@ -4,6 +4,8 @@ import com.typesafe.sbt.pgp.PgpKeys.publishSigned
 enablePlugins(JmhPlugin, ScaladocNoVerificationOfDiagrams)
 disablePlugins(Unidoc, MimaPlugin)
 
+whitesourceIgnore := true
+
 AkkaBuild.defaultSettings
 
 AkkaBuild.dontPublishSettings
```

Comment:
Ah, much neater even, yes!

Is there a more elegant way to make that key available instead of `import sbtwhitesource.WhiteSourcePlugin.autoImport.whitesourceIgnore`?